packages/documentation/src/pages/platform/front-end-standards/security

```diff
@@ -0,0 +1,27 @@
+# Front end security 
```

Comment:
how is CSRF handled?

Reply:
AFAIK we don't. we may have implemented some stuff for the direct deposit changes. I'll follow-up with @ncksllvn